doc/tutorial/alignment_file/index.md

```diff
@@ -252,9 +252,9 @@ With those information do the following:
 Your program should print the following:
 
 ```
-r001 mapped against 0 with 2 gaps in the read sequence and 2 gaps in the reference sequence.
+r001 mapped against 0 with 1 gaps in the read sequence and 2 gaps in the reference sequence.
```

Comment:
Was the old output wrong or was something changed?

Reply:
The old output was wrong. It used the first sequence as the second.